local.toml

```
@@ -11,7 +11,8 @@ title = "gitleaks config"
 	regex = '''\b(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\b'''
 	tags = ["IPv4", "IP", "addresses"]
 	[[rules.whitelist]]
-		regex = '''(169.254.169.254|127.0.0.\d+)'''
+		regex = '''(169.254.169.254|127.0.0.\d+|23.22.13.113)''' # 23.22.13.113 is gsa.ogv
```

Spelling error in the comment:

```suggestion
		regex = '''(169.254.169.254|127.0.0.\d+|23.22.13.113)''' # 23.22.13.113 is gsa.gov
```